app/controllers/discourse_subscriptions/subscribe_controller.rb

```diff
@@ -31,6 +31,18 @@ def index
       end
     end
 
+    def get_contributors
```

Comment:
Probably should name it `contributors` to be consistent with other action names.